src/Cake.Core.Tests/Fixtures/GlobberFixture.cs

```diff
@@ -125,10 +125,10 @@ public Path[] Match(string pattern)
             return Match(pattern, null);
         }
 
-        public Path[] Match(string pattern, Func<IFileSystemInfo, bool> predicate)
+        public Path[] Match(string pattern, Func<IFileSystemInfo, bool> directoryPredicate, Func<IFile, bool> filePredicate = null)
```

Comment:
Keep `Match(string pattern, Func<IFileSystemInfo, bool> predicate)` as an overload calling this method so we don't break binary compatibility.